.azure/pipelines/jobs/iisintegration_job.yml

```
@@ -0,0 +1,26 @@
+
+parameters:
+  agentOs: 'Windows'
+  configuration: 'Release'
+  buildArgs: ''
+  jobName: ''
+
+jobs:
+- template: .vsts-pipelines/templates/project-ci.yml@buildtools
```

I don't think you can mix phases and jobs syntax. You'll have to make this change anywhere "vsts-pipelines" templates are used.

```suggestion
- template: .azure/pipelines/templates/project-ci.yml@buildtools
```